tensorflow_datasets/image_classification/kaokore/kaokore.py

```diff
@@ -0,0 +1,106 @@
+"""kaokore dataset."""
+
+import csv
+import tensorflow.compat.v2 as tf
+import tensorflow_datasets.public_api as tfds
+
+_DESCRIPTION = """
+KaoKore is a novel dataset of face images from Japanese illustrations along with multiple labels for each face, derived from the Collection of Facial Expressions.
+"""
+
+_CITATION = """\
+@misc{tian2020kaokore,
+    title={KaoKore: A Pre-modern Japanese Art Facial Expression Dataset},
+    author={Yingtao Tian and Chikahiko Suzuki and Tarin Clanuwat and Mikel Bober-Irizar and Alex Lamb and Asanobu Kitamoto},
+    year={2020},
+    eprint={2002.08595},
+    archivePrefix={arXiv},
+    primaryClass={cs.CV}
+}
+"""
+
+_URL = 'https://raw.githubusercontent.com/rois-codh/kaokore/9acba43d89f2a32a0b126f1aae3603da6f56345f/dataset_v1.2'
+_IMAGE_SIZE = 256
+_IMAGE_SHAPE = (_IMAGE_SIZE, _IMAGE_SIZE, 3)
+_LABELS = {
+    'gender': ['male', 'female'],
+    'status': ['noble', 'warrior', 'incarnation', 'commoner'],
+}
+
+
+class Kaokore(tfds.core.GeneratorBasedBuilder):
+    """DatasetBuilder for kaokore dataset."""
```

Comment:
Tensorflow follows a slightly modified version of the traditional code style checked by pylint (for instance, two spaces instead of four). Cou[d you format the code according to the style guide as mentioned [here](https://www.tensorflow.org/community/contribute/code_style)